magellan-sample-advanced/src/main/java/com/wealthfront/magellan/sample/advanced/Expedition.kt

```diff
@@ -20,8 +21,13 @@ class Expedition @Inject constructor() : LegacyExpedition<ExpeditionBinding>(
   @set:Inject var navListener: LoggingNavigableListener by lateinitLifecycle()
 
   override fun onCreate(context: Context) {
+    attachToLifecycle(ToolbarHelper)
     app(context).injector().inject(this)
-    navigator.goTo(DogListStep(::goToDetailsScreen))
+  }
+
+  override fun onShow(context: Context, binding: ExpeditionBinding) {
+    ToolbarHelper.init(viewBinding!!.menu, navigator)
```

Comment:
It makes sense to use a static class in the example, but do we want to add a comment explaining that you would usually use DI for this?